### PR TITLE
HAML to ERB Conversion

### PR DESCRIPTION
**Pull request by johnsja**
*Converting HAML to ERB; this will make future changes easier to manage. Developers will no longer need to concern themselves with special syntaxes or precision indenting issues.
*Updating admin/group views and functionality to utilize strong params properly and leverage standard inputs.
*Removing IE8 support as IE8 was EOL'd in January of 2016
*Adding webconsole gem to development group for easier local troubleshooting

### CODE REVIEW

File: app/views/home/index.html.erb [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+    <div class="col-md-6 system-description">
+      <h2><%= @system_info.system_header %></h2>
+      <%= @system_info.system_description.html_safe %>
+    </div>
+  </div>
+  <hr/>
+  <h3>Find more ways to get involved with research!</h3>
+  <div class="row">
+    <div class="col-md-6 rm-link">
+      <h4><a href="https://researchmatch.org?rm=#{@system_info.research_match_campaign}" target:"_blank"><%= image_tag "researchmatch.png" %> Research Match</a></h4>
```
Comment (cdinger) [inline app/views/home/index.html.erb:18]:
```suggestion
      <h4><a href="https://researchmatch.org?rm=<%= @system_info.research_match_campaign %>" target:"_blank"><%= image_tag "researchmatch.png" %> Research Match</a></h4>
```

File: app/views/studies/index.html.erb [hunk 2]
```diff
@@ -0,0 +1,334 @@
...
+    <% if !params[:search].nil? and !params[:search][:category].nil? %>
+      <a class="btn btn-default" href="<%=remove_category_param%>">Search all categories</a>
+      <hr class="tighten"/>
+      <% unless @group.subgroups.empty? %>
+        <p class="subgroup-text"><strong>Suggestions within category "#{@group.group_name}"</strong></p>
+        <div class="subgroups">
+          <% url = request.original_url.encode('utf-8', 'ascii-8bit') %>
+          <% p = request.query_parameters.dup %>
+          <% @group.subgroups.sort_by{ |l| l.name }.each do |subgroup| %>
+            <% p[:search][:q] = subgroup.nam %>
```
Comment (cdinger) [inline app/views/studies/index.html.erb:48]: typo on `subgroup.nam`

File: app/views/studies/index.html.erb [hunk 3]
```diff
@@ -0,0 +1,334 @@
...
+<div class="study-results">
+  <div class="study-results-info">
+    <h3>Search Results <%= 'Within Category "' + @group.group_name + '"' if !params[:search].nil? and !params[:search][:category].nil? %></h3>
+    <p class="matching-message">Here are the studies that match your search criteria. If you are interested in participating, please reach out to the contact listed for the study. If no contact is listed, contact us and we'll help you find the right person.</p>
+
+    <% if !params[:search].nil? and !params[:search][:category].nil? %>
+      <a class="btn btn-default" href="<%=remove_category_param%>">Search all categories</a>
+      <hr class="tighten"/>
+      <% unless @group.subgroups.empty? %>
+        <p class="subgroup-text"><strong>Suggestions within category "#{@group.group_name}"</strong></p>
```
Comment (cdinger) [inline app/views/studies/index.html.erb:43]:
```suggestion
        <p class="subgroup-text"><strong>Suggestions within category "<%= @group.group_name %>"</strong></p>
```

File: app/views/contact/index.html.erb [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+  </div>
+</div>
+
+<% if @system_info.captcha %>
+  <script src="https://www.google.com/recaptcha/api.js?onload=onloadCallback&render=explicit"></script>
+  <script>
+    var myselfWidget;
+    var onloadCallback = function() {
+      myselfWidget = grecaptcha.render('myself-captcha', {
+        'sitekey' : "#{ Recaptcha.configuration.site_key }"
```
Comment (cdinger) [inline app/views/contact/index.html.erb:39]:
```suggestion
        'sitekey' : '<%= Recaptcha.configuration.site_key %>'
```

File: app/views/studies/index.html.erb [hunk 5]
```diff
@@ -0,0 +1,334 @@
+<% if @system_info.captcha %>
+  <script>
+    var myselfWidget;
+    var studyTeamWidget;
+    var onloadCallback = function() {
+      myselfWidget = grecaptcha.render('myself-captcha', {
+        'sitekey' : "#{ Recaptcha.configuration.site_key }"
```
Comment (cdinger) [inline app/views/studies/index.html.erb:7]:
```suggestion
        'sitekey' : '<%= Recaptcha.configuration.site_key %>'
```

File: app/views/studies/index.html.erb [hunk 6]
```diff
@@ -0,0 +1,334 @@
...
+      });
+      studyTeamWidget = grecaptcha.render('study-team-captcha', {
+        'sitekey' : "#{ Recaptcha.configuration.site_key }"
```
Comment (cdinger) [inline app/views/studies/index.html.erb:10]:
```suggestion
        'sitekey' : '<%= Recaptcha.configuration.site_key %>'
```

File: app/views/studies/index.html.erb [hunk 7]
```diff
@@ -0,0 +1,334 @@
...
+      <% end %>
+
+      <div class="trial-cta-options">
+        <div class="btn btn-school btn-email-me" data-toggle='modal' data-target='#email-me-modal' data-title="<%=t.display_title%>" data-trial-id="<%=t.id%>" onclick="track('send', 'event', 'email_me', 'open', '#{t.system_id}')"><i class="fa fa-envelope"></i> Email this study information to me</div>
+        
+        <% unless c.empty? %>
+          <div class="btn btn-school btn-email-study-team" data-toggle='modal' data-target='#contact-study-team-modal' data-email="<%=c.first[:email]%>" data-trial-id="<%=t.id%>" onclick="track('send', 'event', 'email_study_team', 'open', '#{t.system_id}')"><i class="fa fa-envelope"></i> Contact the study team</div>
+        <% end %>
+
+        <a class="btn btn-school btn-more-info" href="https://www.clinicaltrials.gov/ct2/show/study/#{t.system_id}" onclick="track('send', 'event', 'ctgov', 'click', '#{t.system_id}')" target="_blank"><i class="fa fa-info-circle"></i> See more information</a>
```
Comment (cdinger) [inline app/views/studies/index.html.erb:213]:
```suggestion
        <a class="btn btn-school btn-more-info" href="https://www.clinicaltrials.gov/ct2/show/study/<%= t.system_id %>" onclick="track('send', 'event', 'ctgov', 'click', '<%= t.system_id %>')" target="_blank"><i class="fa fa-info-circle"></i> See more information</a>
```

File: app/views/studies/index.html.erb [hunk 8]
```diff
@@ -0,0 +1,334 @@
...
+
+      <% if t.respond_to?(:disease_sites) && !t.disease_sites.blank? %>
+        <div class="field nomargin">
+          <label class="single">Disease Sites:</label>
+          <%= t.disease_sites.map {|x| x.disease_site_name }.join('; ') %>
+        </div>
+      <% end %>
+
+      <div class="trial-cta-options">
+        <div class="btn btn-school btn-email-me" data-toggle='modal' data-target='#email-me-modal' data-title="<%=t.display_title%>" data-trial-id="<%=t.id%>" onclick="track('send', 'event', 'email_me', 'open', '#{t.system_id}')"><i class="fa fa-envelope"></i> Email this study information to me</div>
```
Comment (cdinger) [inline app/views/studies/index.html.erb:207]:
```suggestion
        <div class="btn btn-school btn-email-me" data-toggle='modal' data-target='#email-me-modal' data-title="<%=t.display_title%>" data-trial-id="<%=t.id%>" onclick="track('send', 'event', 'email_me', 'open', '<%= t.system_id %>')"><i class="fa fa-envelope"></i> Email this study information to me</div>
```

File: app/views/studies/index.html.erb (hunk 2 above)
Comment (johnsja) [reply app/views/studies/index.html.erb]: Fixed. Good catch @cdinger

**Pull request merged**